features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -430,7 +434,8 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
             regEx:  values?.regularExpression !== undefined ? values.regularExpression?.toString() : claim?.regEx,
             required: values?.required !== undefined && !values?.readOnly ? !!values.required : false,
             supportedByDefault: values?.supportedByDefault !== undefined
-                ? !!values.supportedByDefault : claim?.supportedByDefault
+                ? !!values.supportedByDefault : claim?.supportedByDefault,
```

Comment:
Can be improved as,
`supportedByDefault: values?.supportedByDefault ?? claim?.supportedByDefault`

Reply:
No changes were made; only a comma was added. Discussed with Jayashajthi and concluded not to make any amendments.